icons/angular/angular-plain.svg

```diff
@@ -1,6 +1 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 128 128" version="1.1">
-  <g id="surface1">
-    <path style=" stroke:none;fill-rule:nonzero;fill:rgb(0%,0%,0%);fill-opacity:1;" d="M 55.296875 69.324219 L 72.703125 69.324219 L 64 48.382812 Z M 55.296875 69.324219 M 64 15.359375 L 16.332031 32.359375 L 23.601562 95.386719 L 64 117.761719 L 104.398438 95.386719 L 111.667969 32.359375 Z M 93.746094 93.492188 L 82.636719 93.492188 L 76.644531 78.539062 L 51.355469 78.539062 L 45.363281 93.492188 L 34.253906 93.492188 L 64 26.675781 Z M 93.746094 93.492188 "/>
-  </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 128 128"><path d="m78.36 0 41.77 89.9 4.34-68.57L78.36 0M38.87 94.47l-6.42 15.54L64 128l31.55-17.99-6.42-15.54H38.87M80.53 74.3 64 34.11 47.47 74.3h33.06m-77-52.97L7.87 89.9 49.64 0 3.53 21.33"/></svg>
```

Comment:
Much better!
It can be optimized even further by removing the data-layer:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="m78.36 0 41.77 89.9 4.34-68.57L78.36 0M38.87 94.47l-6.42 15.54L64 128l31.55-17.99-6.42-15.54H38.87M80.53 74.3 64 34.11 47.47 74.3h33.06m-77-52.97L7.87 89.9 49.64 0 3.53 21.33"/></svg>
```